src/Illuminate/Console/Scheduling/ManagesFrequencies.php

```diff
@@ -2,20 +2,23 @@
 
 namespace Illuminate\Console\Scheduling;
 
+use Illuminate\Console\Enums\ScheduleOn;
 use Illuminate\Support\Carbon;
 use InvalidArgumentException;
 
+use function Illuminate\Support\enum_value;
+
 trait ManagesFrequencies
 {
     /**
      * The Cron expression representing the event's frequency.
      *
-     * @param  string  $expression
+     * @param  string|\Illuminate\Console\Enums\ScheduleOn  $expression
```

Comment:
Is this method even supposed to accept single integers like `->cron(3)`? I thought it's intended for those cron strings like `0 15 10 ? * 6L 2002-2005`.

Reply:
I decided to add validation in a separate PR: #53971